quirks/change-password-URLs.json

```diff
@@ -1,5 +1,6 @@
 {
     "500px.com": "https://web.500px.com/settings/account/security",
+    "acorns.com" "https://app.acorns.com/settings/change-password,
```

Comment:
You need a closing quote on this line. :)

Reply:
Thanks, @rmondello.

I did not realize i made such a silly mistake 👎 
Correct it

Reply:
No worries! Mistakes happen!